tilt/nvidia-driver-daemonset.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       labels:
         app: nvidia-driver-daemonset
     spec:
+      nodeSelector:
+        nvidia.com/gpu.present: "true"
```

Comment:
Same comment as earlier, we will need to remove this, so that pod runs on all worker nodes and not just the ones created by kwok

Reply:
yep. done